src/prisma/schema.prisma

```diff
@@ -3261,23 +3264,40 @@ model WeeklyClaims {
 }
 
 model TokensReceipt {
-  id          String              @id @default(uuid()) @db.Uuid
-  createdAt   DateTime            @default(now())
-  value       Int
-  claimedAt   DateTime?
-  eventId     String              @db.Uuid
-  event       BuilderEvent        @relation(fields: [eventId], references: [id], onDelete: Cascade)
-  recipientId String?             @db.Uuid
-  recipient   Scout?              @relation(fields: [recipientId], references: [id], onDelete: Cascade, name: "tokenReceiptRecipient")
-  senderId    String?             @db.Uuid
-  sender      Scout?              @relation(fields: [senderId], references: [id], onDelete: Cascade, name: "tokenReceiptSender")
-  activities  ScoutGameActivity[]
+  id            String              @id @default(uuid()) @db.Uuid
+  createdAt     DateTime            @default(now())
+  value         Int
+  claimedAt     DateTime?
+  eventId       String              @db.Uuid
+  event         BuilderEvent        @relation(fields: [eventId], references: [id], onDelete: Cascade)
+  walletAddress String
+  recipientId   String?             @db.Uuid
+  recipient     Scout?              @relation(fields: [recipientId], references: [id], onDelete: Cascade, name: "tokenReceiptRecipient")
+  senderId      String?             @db.Uuid
+  sender        Scout?              @relation(fields: [senderId], references: [id], onDelete: Cascade, name: "tokenReceiptSender")
```

Comment:
replace receipitnId/senderId with recipientWallet and senderWallet and connect them to ScoutWallet